src/views/CurrentReservations.vue

```diff
@@ -42,10 +42,10 @@ export default {
   },
   methods: {
     // sends to the user to the map to edit a reservation
-    viewReservation(fid) {
+    viewReservation(id) {
       this.$router.push({
         name: 'ReserveEdit',
-        params: { activeStreetFid: fid, editmode: 'edit' },
+        params: { activeStreetFid: id, editmode: 'edit' },
```

Comment:
Should this param be activeStreetID instead of FID?

Reply:
Yeah, missed that change